test/integration_test.go

```diff
@@ -225,21 +223,13 @@ func mustMakeOperators(t *testing.T, cst *coremock.ChainDataMock, logger common.
 		logPath := fmt.Sprintf("testdata/%v/log", idStr)
 
 		err := os.RemoveAll(dbPath)
-		if err != nil {
-			t.Fatal(err)
-		}
+		assert.NoError(t, err)
 		err = os.RemoveAll(logPath)
-		if err != nil {
-			t.Fatal(err)
-		}
+		assert.NoError(t, err)
 		err = os.MkdirAll(dbPath, os.ModePerm)
-		if err != nil {
-			t.Fatal(err)
-		}
+		assert.NoError(t, err)
```

Comment:
If this errors, won't it cause the test to essentially fail? I feel like aborting the test might make sense in cases like this.

Reply:
Ah, that was why we were using `Fatal`. I'll just revert these changes